src/utils/release-notes/index.ts

```diff
@@ -0,0 +1,50 @@
+import * as vscode from "vscode";
```

Comment:
Do we need to add license header in the sources?

Reply:
Nice reminder! I'll add in a separate PR.